protected/humhub/modules/topic/jobs/ConvertTopicsToGlobalJob.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace humhub\modules\topic\jobs;
+
+use humhub\modules\queue\ActiveJob;
+use humhub\modules\queue\interfaces\ExclusiveJobInterface;
+use humhub\modules\topic\models\Topic;
+
+class ConvertTopicsToGlobalJob extends ActiveJob implements ExclusiveJobInterface
+{
+    public function getExclusiveJobId()
+    {
+        return 'module.topics.global-conversion';
+    }
+
+    public function run()
+    {
+        Topic::convertToGlobal();
```

Comment:
Sometimes we're calling this static method directly, sometimes via this job?
Not sure here...

Reply:
Ah ok understand. I think we'll remove the "convert ALL topics" for now.